projects/examples/src/components/subscription/subscription-tracker.examples.module.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright 2020 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { NgModule } from '@angular/core';
+import { Documentation } from '@vcd/ui-doc-lib';
+import { SubscriptionTrackerMixinExampleModule } from './subscription-tracker-mixin.example.module';
+import { SubscriptionTrackerMixinExampleComponent } from './subscription-tracker-mixin.example.component';
+import { SubscriptionTracker } from '@vcd/ui-components';
+
+Documentation.registerDocumentationEntry({
+    component: SubscriptionTracker,
+    displayName: 'Subscription Tracker',
+    urlSegment: 'subscriptionTracker',
+    examples: [
+        {
+            component: SubscriptionTrackerMixinExampleComponent,
+            forComponent: null,
+            title: 'Example to show how to use the subscription tracker',
```

Comment:
Let's avoid "example" in the title and try to describe the feature: "Automatically destroy subscriptions when components are destroyed"